build.sh

```diff
@@ -18,7 +18,7 @@ ARGS=$*
 # scripts, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libcuvs python rust docs tests bench-ann examples --uninstall  -v -g -n --compile-static-lib --allgpuarch --no-nvtx --show_depr_warn --incl-cache-stats --time -h"
+VALIDARGS="clean libcuvs python rust docs tests bench-ann examples --uninstall  -v -g -n --compile-static-lib --allgpuarch --mg_algos_off --no-nvtx --show_depr_warn --incl-cache-stats --time -h"
```

Comment:
I thought that it is an important feature that we would like to have by default. Even though it can be interesting to have an option to get a smaller build with fewer dependencies. Do we want to change this?

Reply:
I think maybe we should consider not having this on by default, but we should make sure we add `--mg` (I propose we rename it to be simpler) to all the conda recipes. Please also add this to the docs both in the "help" for this file and in the build and install docs (docs/source/build.md).